src/frontend/screens/Onboarding/welcome/index.module.scss

```diff
@@ -32,8 +39,38 @@
 .welcomeBodyTextContainer a:hover {
   color: var(--color-primary-300);
   transition: 0.2s;
+  font-size: var(--font-size-md);
+}
+
+.text {
+  font-weight: 400;
 }
```

Comment:
We pretty much never have to set the specific typography in a component since we can reuse the styles from hp/ui `src\styles\designSystem\_typography.scss`. Do we need a new style or to update an existing style in the global stylesheet? If we make custom type sizing, weights, fonts, or line-heights on a per component basis, the typography system quickly become unmaintainable.

Reply:
I had to create a class to style a small piece of text differently because it was inheriting bold from its parent. I'll see how to change it.